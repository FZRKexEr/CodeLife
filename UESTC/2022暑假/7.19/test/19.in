10
dndbixwitnjyktvulftecegzwrehamknjyazxysifpizvxgsjhctibyfmjrrbdloqqwkdrgpqcgksfgbfcoqcybkbcsqzkxcmtby
10
orwtwgdnkgmrcduiwobummkgombdumthysdyxwftnulmjzkurjzqjzxarvcakenolbuveidznuaeihsqzrwhrcjtemmoxtzvnxxt
6
hmfwwrlyejikupdqaqntvfatzstcpfhqoyiwzticedpsqszowteqbtshsmrgyuvaolqanjssyardthqwpcgiqgbapoxeclrkqrms
2
eidgfgcviflhvcwcnoeeeiednkyrbgpecqhndvbezdcqenfvzklenjyikapsrgaqmypwizqqavdkccrsigkmbexyqsxljcqfovqe
7
uqudueluuahzxrblfogyjdtcrvnqxiywhlpobwmmredqqudkmgbxxkgejhypghbbaqvjhzvwhjhwcrynebvqkpccqqivrnmmplea
1
uwvvoyeyfitwkkqbyxcqynhoxfthxqysnqnyppthwoluhtzcsxtlaouezgfqkrxibaldfnlxddpbbdaqdnlixxlwelzhehyhezpw
1
ppyarhvvasmkouremrsywsayweuarcuaeanfsuhffvhiqiwksncfoznecullcnjcybzygipdxhpfjobyqfzapjwnarenajedelmk
4
glqupsywzzwdkcliquwdycvljrpouojvchflqlcmvckhymortjhgpccrfndggqynpmcrifjqokzscwcmbfaywwcdppvoevcmouii
6
xdmkrhtbpdyxymlhovfvvifzktfqtxjnkigbcvskywwjhobwpdbsnnbbvqsfzubtjmzvodgvsfmerzgqwnxvcvoqthavewckasmw
3
uhefznyulmdhcdmfmbhiyhspfgjygzzjeuprjtztddrunpxaccihwqyjfxqzgnutbihgaizyuidavzpixiznmoesnbrloydsmhax
7
