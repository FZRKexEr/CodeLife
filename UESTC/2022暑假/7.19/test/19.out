1024
65536
65980984
16384
882499718
882499718
33554432
65536
603979768
16384
